1_OCaml/TP/TP1.ipynb

```diff
@@ -20,6 +20,78 @@
     "Puis utilisez ces fonctions sur des arguments de votre choix."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    let p2 = [x2,y2] in
</pre><p>Parenthèses et pas crochets pour définir un point </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>